src/components/simulation/CustomContent.vue

```diff
@@ -0,0 +1,40 @@
+<script setup lang="ts">
+import step2 from '@/assets/illustrations/Steps.jpg'
+import validation from '@/assets/icons/interface-validation.svg'
+import deleted from '@/assets/icons/interface-delete.svg'
+
+const props = defineProps({
+  step: {
+    type: Number,
+    required: true,
+  },
+})
+</script>
+
+<template>
+  <div v-if="props.step === 2">
```

Comment:
Can you add this if directly in file StepSunshineView please
Like that: 
![image](https://user-images.githubusercontent.com/121813008/224686843-4708546d-2d43-4a8d-afd8-51cacf7c2142.png)